datumaro/util/image.py

```diff
@@ -380,3 +382,32 @@ def save(self, path):
                 f.write(self.get_bytes())
         else:
             save_image(path, self.data)
+
+ImageMeta = Dict[str, Tuple[int, int]]
+
+DEFAULT_IMAGE_META_FILE_NAME = 'images.meta'
+
+def load_image_meta_file(image_meta_path: str) -> ImageMeta:
+    """
+    Loads image metadata from a file with the following format:
+
+        <image name 1> <height 1> <width 1>
+        <image name 2> <height 2> <width 2>
+        ...
+
+    This can be useful to support datasets in which image dimensions are
+    required to interpret annotations.
+    """
+    assert isinstance(image_meta_path, str)
+
+    if not osp.isfile(image_meta_path):
+        raise Exception("Can't read image meta file '%s'" % image_meta_path)
+
+    image_meta = {}
+
+    with open(image_meta_path, encoding='utf-8') as f:
+        for line in f:
+            image_name, h, w = line.strip().rsplit(maxsplit=2)
+            image_meta[image_name] = (int(h), int(w))
```

Comment:
Probably, we need to skip empty (and maybe commented) lines here too.
```suggestion
            line = line.strip()
            if not line or line.startswith('#'):
                continue
            
            image_name, h, w = line.strip().rsplit(maxsplit=2)
            image_meta[image_name] = (int(h), int(w))
```

Reply:
I implemented an approach based on `shlex.split` - that handles comments and allows other fields to be added in the future. What do you think?

Reply:
Maybe it is a good solution, but I find it hard to understand the grammar it describes from the official description (both in POSIX and non-POSIX modes), so I'm having troubles with predicting the possible consequences.

Reply:
It's just (very simplified) shell syntax. The main consequence is that file names with spaces in them have to be quoted.

Reply:
Consider updating few other formats to use `shlex`, such syntax is used in few extra dataset files from datumaro - VOC, CamVid and several others.

Reply:
I will take a look later, but right now I want to get back to Open Images.

FWIW, I found four likely candidates for updating:

```console
$ rg -Fl "line[0] == '#'"
datumaro/plugins/cityscapes_format.py
datumaro/plugins/voc_format/format.py
datumaro/plugins/camvid_format.py
datumaro/plugins/kitti_format/format.py
```

Not sure about LFW.